tests/commonTemplates_test.go

```diff
@@ -208,6 +211,42 @@ var _ = Describe("Common templates", func() {
 	})
 
 	Context("resource change", func() {
+		Context("namespace change", func() {
+			BeforeEach(func() {
+				Eventually(func() error {
+					namespaceObj := &v1.Namespace{ObjectMeta: metav1.ObjectMeta{Name: strategy.GetNewTemplatesNamespace()}}
+					return apiClient.Create(ctx, namespaceObj)
+				}, timeout, 10*time.Second).ShouldNot(HaveOccurred())
+				strategy.SkipSspUpdateTestsIfNeeded()
```

Comment:
This call should be at the top of `BeforeEach()`. When the test is skipped, the namespace does not need to be created.